src/Simplex/Chat/Types.hs

```diff
@@ -1527,3 +1532,24 @@ newtype JVersionRange = JVersionRange {fromJVersionRange :: VersionRange} derivi
 instance ToJSON JVersionRange where
   toJSON (JVersionRange (VersionRange minV maxV)) = J.object ["minVersion" .= minV, "maxVersion" .= maxV]
   toEncoding (JVersionRange (VersionRange minV maxV)) = J.pairs $ "minVersion" .= minV <> "maxVersion" .= maxV
+
+newtype RemoteHostId = RemoteHostId Int
+  deriving stock (Show)
+  deriving newtype (Eq, Ord, FromJSON, ToJSON, FromField, ToField, StrEncoding)
+
+pattern LOCAL_HOST_ID :: Maybe RemoteHostId
+pattern LOCAL_HOST_ID = Nothing
+{-# COMPLETE LOCAL_HOST_ID, Just #-}
```

Comment:
why can't we just use Nothing? seems confusing rather than helpful.

Reply:
We don't define what "Local host" means, and it kind of obvious that if remoteHostId is Nothing then it's not remote.